### ISSUE TITLE
using the orca screen reader, notifications are not readable when arrowing up and down

### ISSUE THREAD

**Issue by mhussaincov93** [AyatanaIndicators/ayatana-indicator-notifications#20]
hi there,
when arrowing up and down in the list of notifications, the notifications are not read out by orca the screen reader.
the only item that is read is the menu item named doo not disterb.
it is the only item that is interactable 
that is to say, that I am able to press enter on it.
there are other items in the list, because orca stayes silent when arrowing up and down the list, but does not speak till you hit the doo not desterb item.

stats that you may wish to have.
running ubuntu-mate development branch.
orca v40.0
even when running on the regular release of ubuntu mate this issue persisted.
I am blind.
I was redirected here by martin wimpress from ubuntu-mate.
thanks for your help,
Majid

**Comment by sunweaver** [comment]
@mhussaincov93: you issue has been addressed in AyatanaIndicators/ayatana-ido#33, for an easy testing, you'll have to wait for Ubuntu MATE 21.10. I'll have to check whether I can get the fixed into Debian 11 (aka bullseye).

@tari01: we can close this issue now, can't we?

**Comment by tari01** [comment]
Resolved via https://github.com/AyatanaIndicators/ayatana-ido/pull/33

**Comment by sunweaver** [comment]
Forwarded to Debian BTS as https://bugs.debian.org/988680
